attestation-service/README.md

```diff
@@ -1,31 +1,60 @@
+The AS was designed to be used with the [KBS](../kbs) for Confidential Containers,
+but it can be used in a wide variety of situations.
+The AS can be used anytime TEE evidence needs to be validated.
+
+Today, the AS can validate evidence from the following TEEs:
+- Intel TDX
+- Intel SGX
+- AMD SEV-SNP
+- ARM CCA
+- Hygon CSV
+- Intel TDX with vTPM on Azure
+- AMD SEV-SNP with vTPM on Azure
+
+# Overview
+```
+                                      ┌───────────────────────┐
+┌───────────────────────┐ Evidence    │  Attestation Service  │
+│                       ├────────────►│                       │
+│ Verification Demander │             │ ┌────────┐ ┌──────────┴───────┐
+│    (Such as KBS)      │             │ │ Policy │ │ Reference Value  │◄───Reference Value
+│                       │◄────────────┤ │ Engine │ │ Provider Service │
+└───────────────────────┘ Attestation │ └────────┘ └──────────┬───────┘
+                        Results Token │                       │
+                                      │ ┌───────────────────┐ │
+                                      │ │  Verifier Drivers │ │
+                                      │ └───────────────────┘ │
+                                      │                       │
+                                      └───────────────────────┘
+```
 
-Those two steps are accomplished by respectively one of the [Verifier Drivers](#verifier-drivers) and the AS [Policy Engine](#policy-engine). The policy engine can be customized with different policy configurations.
+The AS has a simple API. It receives attestation evidence and returns an attestation token
+containing the results of a two-step verification process.
+The AS can be consumed directly as a Rust crate (library) or built as a standalone service,
+exposing a REST or gRPC API.
+In Confidential Containers, the client of the AS is the KBS, but the evidence originates
+from the Attestation Agent inside the guest.
 
-In addition, CoCo-AS can also run independently as a remote attestation service which receives TEE evidence and returns the verification results as a token.
+The AS has a two-step verification process.
 
-The AS can be built as a library (i.e. a Rust crate) by other confidential computing resources providers, like for example the KBS.
-It can also run as a standalone binary, which exposes APIs in gRPC way.
+1. Verify the format and provenance of evidence itself (i.e. check the signature of the evidence).
+2. Evaluate the claims presented by the evidence (i.e. check that measurements are what the client expects).
 
-# Components
+The first step is accomplished by one of the platform-specific [Verifier Drivers](#verifier-drivers).
+The second step is driven by the [Policy Engine](#policy-engine) with help from the [RVPS](#reference-value-provider-service).
 
+# Usage
 ## Library
 
 The AS can be built and imported as a Rust crate into any project providing attestation services.
```

Comment:
Should there be any mention of CoCo AS and Amber/IntelTrustAuthority AS?

Reply:
ITA is supported in the KBS as an alternative to the AS. The AS itself doesn't have any support for it. We do have some support for veraison in the AS, but it's just part of the CCA verifier.